docs/cosmos-stack-overview.mdx

```diff
@@ -0,0 +1,503 @@
+
+        <TopicCard
+          title="Cosmos EVM"
+          description="EVM compatibility layer with Solidity support and access to Cosmos SDK features"
+          color="#aaaaaa"
+          links={[
+            { text: 'EVM Overview', href: '/docs/evm/next/documentation/overview' },
+            { text: 'Build a Chain', href: '/docs/evm/next/documentation/getting-started/build-a-chain/overview' },
+            { text: 'Deploy Contracts', href: '/docs/evm/next/documentation/smart-contracts/introduction' },
+            { text: 'Precompiles', href: '/docs/evm/next/documentation/smart-contracts/precompiles/overview' },
+            { text: 'EVM Compatibility', href: '/docs/evm/next/documentation/evm-compatibility/overview' },
+            { text: 'JSON-RPC API', href: '/docs/evm/next/api-reference/ethereum-json-rpc/index' },
+          ]}
+        />
+
+        <TopicCard
+          title="Development Tools"
+          description="Essential tools and frameworks for building and deploying on the Cosmos stack"
+          color="#777777"
+          links={[
+            { text: 'Hardhat Setup', href: '/docs/evm/next/documentation/getting-started/tooling-and-resources/hardhat' },
+            { text: 'Foundry Setup', href: '/docs/evm/next/documentation/getting-started/tooling-and-resources/foundry' },
+            { text: 'Block Explorers', href: '/docs/evm/next/documentation/getting-started/tooling-and-resources/block-explorers' },
+            { text: 'Wallet Integration', href: '/docs/evm/next/documentation/getting-started/tooling-and-resources/wallet-integration' },
+            { text: 'Testing & Fuzzing', href: '/docs/evm/next/documentation/getting-started/tooling-and-resources/testing-and-fuzzing' },
+          ]}
+        />
+
+        <TopicCard
+          title="Getting Started"
+          description="Quick start guides and tutorials to begin building on the Cosmos stack"
+          color="#666666"
+          links={[
+            { text: 'Quick Start', href: '/docs/evm/next/documentation/getting-started/index' },
+            { text: 'Local Node Setup', href: '/docs/evm/next/documentation/getting-started/local-node-setup' },
+            { text: 'Development Environment', href: '/docs/evm/next/documentation/getting-started/tooling-and-resources/development-environment' },
+            { text: 'FAQ', href: '/docs/evm/next/documentation/getting-started/faq' },
+          ]}
+        />
+      </TopicCardGrid>
+    </section>
+
+    {/* About Sections */}
+    <section style={{ marginBottom: '80px', paddingBottom: '80px', borderBottom: '1px solid #cccccc10' }}>
+
+      <div style={{
+        background: '#0E0E0E',
+        border: '1px solid #cccccc20',
+        borderRadius: '12px',
+        padding: '40px',
+        marginBottom: '24px',
+      }}>
+        <h3 style={{
+          fontSize: '24px',
+          fontWeight: '600',
+          marginBottom: '16px',
+          color: '#F1F1F1',
+        }}>
+          About CometBFT
```

Comment:
Can we make each "About" component collapsable? Where you expand to see the text? (sounds like bad design tho..)

Can you order Cosmos SDK, Cosmos EVM, IBC, CometBFT

Reply:
yeah I think there is a better element we could use for these im just not completely sure what it would be. The idea (not my idea) was to keep this very simple like some specific old Linux webpage from 2001 lol